package.json

```diff
@@ -7,7 +7,9 @@
     "clean": "lerna clean && lerna run clean",
     "bootstrap": "lerna bootstrap",
     "publish": "npm run build && lerna publish",
-    "build": "lerna run build",
+    "build": "npm run build:cjs && npm run build:esm",
```

Comment:
note that there is no prepublishOnly scripts here... so it is "manual", though we do have build && lerna publish to smooth this over for us